.github/workflows/qemu-jobs.yml

```diff
@@ -0,0 +1,193 @@
+name: Build and check tests for QFlex enabled and base QEMU
+
+on: 
+  push:
+    branches:
+      - master
+      - dev
+  pull_request:
+    branches:
+      - master
+      - dev
+  workflow_dispatch:
+
+jobs:
+  build_and_check_job_definition: # native_build_job_definition:
+    runs-on: ubuntu-latest
+    name: Configure, build and check QEMU # native_build_job_definition
+    strategy:
+      matrix:
+        include:
+          - # NOTE: --enable-fdt=system --enable-slirp=system was removed
+            build-name: Build for Ubuntu 20.04 + check # build-system-ubuntu
+            targets: aarch64-softmmu
+            make-check-args: check
+          - build-name: Build for user-mode + check-tcg # build-user
+            targets: aarch64-linux-user
+            configure-args: --disable-tools --disable-system
+            make-check-args: check-tcg
+          - build-name: Build for system-mode + check-tcg # build-some-softmmu
+            targets: aarch64-softmmu
+            configure-args: --disable-tools --enable-debug
+            make-check-args: check-tcg
+#          - build-name: build-user-plugins
+#            targets: aarch64-linux-user
+#            configure-args: --disable-tools --disable-system --enable-plugins --enable-debug-tcg
+#            make-check-args: check-tcg
+          - build-name: Build for system-mode with plugins enabled + check-tcg # build-some-softmmu-plugins
+            targets: aarch64-softmmu
+            configure-args: --disable-tools --disable-user --enable-plugins --enable-debug-tcg
+            make-check-args: check-tcg
```

Comment:
This is the exact same thing as the above two.

The difference is that it enables more configuration flags (e.g., plugins and debug-tcg) but looking at the output, still no tests run.

Reply:
Resolved in https://github.com/parsa-epfl/qemu/pull/62#issuecomment-882524912